series-tmdb.en.yaml

```diff
@@ -2912,4 +2912,29 @@ entries:
     seasons:
       - season: 1
         anilist-id: 159831
+        
+  - title: 'Initial D'
+    guid: plex://show/5d9c08567b5c2e001e6598a8
+    seasons:
+      - season: 1
+        anilist-id: 185
+      - season: 2
+        anilist-id: 186
+      - season: 3
```

Comment:
TMDB is missing 3rd stage because it's a movie 

Remove this list entry and decrease the season numbers for the rest so the last season is season 5.

Reply:
You can add a TMDB movie mapping for https://www.themoviedb.org/movie/80741-d-third-stage if you want